plugins/enthusiast-model-openai/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "enthusiast-model-openai"
-version = "1.0.0"
+version = "1.1.0"
```

Comment:
How about we don't bump the package versions just yet - we can do that later, once we verify the interfaces work fine with the server side